sdk/include/reactos/idl/svcctl.idl

```diff
@@ -305,68 +305,68 @@ typedef struct _STRING_PTRSW {
 interface svcctl
 {
     /* Function 0 */
-    DWORD RCloseServiceHandle(
+    DWORD __stdcall RCloseServiceHandle(
```

Comment:
I always thought that idl files defaulted to __stdcall?

Reply:
We got cdecl vs stdcall issues here.

Reply:
Not sure if we mentioned the idls specifically when talking about this, so just in case: these also look dubious.